src/App.js

```diff
@@ -1,31 +1,54 @@
 import './App.css';
+import { Routes, Route } from 'react-router-dom'
+import Header from './Header';
+import Form from './Form.js';
+import DisplayCurrentWeather from './DisplayCurrentWeather';
+import DisplayHourlyWeather from './DisplayHourlyWeather';
+import Display10DaysWeather from './Display10DaysWeather';
+import ErrorPage from './ErrorPage';
+import Footer from './Footer.js';
 
 function App() {
   return (
     <div className="App">
 
+      <Routes>
```

Comment:
It might be better to use router under form component instead of main app.js.
We will merge into main for now and work on axios call first.
let's see what types of data we get back and change how we want to set up the router along the way.